src/ProductSync.php

```diff
@@ -143,6 +143,64 @@ public static function is_product_sync_enabled() {
 		return (bool) $domain_verified && $tracking_enabled && Pinterest_For_Woocommerce()::get_setting( 'product_sync_enabled' );
 	}
 
+	/**
+	 * Get the proper extra_info for the feed status.
+	 *
+	 * @return string
+	 */
+	public static function get_feed_status_extra_info() {
+
+		if ( self::is_product_sync_enabled() ) {
+			return '';
+		}
+
+		if ( ! Pinterest_For_Woocommerce()::is_domain_verified() ) {
+			return sprintf(
+				/* Translators: %1$s The URL of the connection page */
```

Comment:
Minor: for this and the below `translators` comments, you can just use `1:` rather than `%1$s`.

Also note that "translators" should be all lowercase.